From norihiko.tomiyama@ctc-g.co.jp Wed Apr  4 18:06:17 2007
From: Norihiko Tomiyama <norihiko.tomiyama@ctc-g.co.jp>
Date: Thu, 05 Apr 2007 10:05:40 +0900
Subject: [PATCH]Adding PID of SHARP S01SH for ipaq.c
To: gregkh@suse.de
Message-ID: <46144B64.1050907@ctc-g.co.jp>

From: Norihiko Tomiyama <norihiko.tomiyama@ctc-g.co.jp>

I write a patch adding support "SHARP EMONE(S01SH)" device for ipaq.c.
EMONE is a PDA with built-in HSDPA function.



From: Norihiko Tomiyama <norihiko.tomiyama@ctc-g.co.jp>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/serial/ipaq.c |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/usb/serial/ipaq.c
+++ b/drivers/usb/serial/ipaq.c
@@ -255,6 +255,7 @@ static struct usb_device_id ipaq_id_tabl
 	{ USB_DEVICE(0x04DD, 0x9102) }, /* SHARP WS003SH USB Modem */
 	{ USB_DEVICE(0x04DD, 0x9121) }, /* SHARP WS004SH USB Modem */
 	{ USB_DEVICE(0x04DD, 0x9123) }, /* SHARP WS007SH USB Modem */
+	{ USB_DEVICE(0x04DD, 0x9151) }, /* SHARP S01SH USB Modem */
 	{ USB_DEVICE(0x04E8, 0x5F00) }, /* Samsung NEXiO USB Sync */
 	{ USB_DEVICE(0x04E8, 0x5F01) }, /* Samsung NEXiO USB Sync */
 	{ USB_DEVICE(0x04E8, 0x5F02) }, /* Samsung NEXiO USB Sync */
